cmd/snap-update-ns/utils_test.go

```diff
@@ -45,6 +45,11 @@ type utilsSuite struct {
 
 var _ = Suite(&utilsSuite{})
 
+const (
+	// Ext4Magic is the equivalent of EXT4_SUPER_MAGIC
+	Ext4Magic = 0xef53
```

Comment:
interesting, so it was only used in tests?

Reply:
It would seem to make sense to leave it where it was even if it's unused, since it was together with other similar constants; maybe just add the proper nolint comment to have it ignored?

Reply:
Yes, only in tests (all in one package). @mardy that's a fair point, I changed it back :slightly_smiling_face: thanks